apps/common/messengers/RackHDRestAPIv2_0.js

```diff
@@ -0,0 +1,36 @@
+// Copyright 2015, EMC, Inc.
+
+'use strict';
```

Comment:
Just started happening with the latest eslint, I think I need to fix my eslintrc file